frontend/src/metabase/redux/embed.ts

```diff
@@ -18,42 +16,50 @@ export const DEFAULT_EMBED_OPTIONS = {
   action_buttons: true,
 } as const;
 
-export const SET_INITIAL_URL_OPTIONS = "metabase/embed/SET_INITIAL_URL_OPTIONS";
-export const setInitialUrlOptions = createAction(
-  SET_INITIAL_URL_OPTIONS,
-  ({ search, hash }: { search: string; hash: string }) => {
-    return {
-      ...parseSearchOptions(search),
-      ...parseHashOptions(hash),
-    };
+const allowedEmbedOptions = Object.keys(DEFAULT_EMBED_OPTIONS);
+
+export const urlParameterToBoolean = (
```

Comment:
Thanks, I added some basic unit tests to the reducer so that we can test the end results.
I didn't test the things that we don't make explicit (like getting the last value if we have more than value with the same name in the url) as I don't want people to believe that's the intended behaviour (I actually don't think we have one, I mostly made that code to make TS happy without lying to him with a `as string`)

Reply:
I see, testing the reducer itself sounds good! Thanks for adding the tests 🙏🏻